README.md

```diff
@@ -0,0 +1,13 @@
+# Roots Check in Tool
+
+### Run locally
+
+Clone repo: `git clone git@github.com:damwhit/check_in_tool.git`
+
+Bundle dependencies: `bundle`
+
+Establish database: `rake db:setup`
+
```

Comment:
I think this is going end up looking more like `rake db:{create,migrate}` if someone is coming into a clean setup.